ext/net/ops_tls.rs

```diff
@@ -224,10 +246,53 @@ where
     .unwrap();
   Ok(deno_core::cppgc::make_cppgc_object(
     scope,
-    TlsKeys::Static(TlsKey(cert, key)),
+    TlsKeysHolder::from(TlsKeys::Static(TlsKey(cert, key))),
   ))
 }
 
+#[op2]
+pub fn op_tls_cert_resolver_create<'s>(
```

Comment:
No `#[serde]` 👀

Reply:
Yup, we just create a v8::Array of two cppgc objects.

Reply:
Ah sweet, so we can actually do similar thing if we need to return cppgc object and serde struct in the same call 👍